source/includes/api-reference/_api-reference.html.md

```diff
@@ -0,0 +1,347 @@
+
+| Parameter | Description |
+| --- | --- |
+| id | Numerical id of the BRAIN |
+| name | Name of the BRAIN |
+| url | URL of the BRAIN |
+| status | Name of the user |
+
+## GET BRAIN status
+
+> Example Response
+
+```json
+{
+   "id": 1
+   "name": "mybrain1",
+   "simulators": [{
+       "name": "breakout",
+       "objectives": ["score", "ball_location_distance"]
+   }],
+   "latest": "/v1/megan/mybrain/11"
+}
+```
+
+```protobuf
+// There is no protobuf version of this response.
+```
+
+Get information about a BRAIN.
+
+### Request
+
+`GET /v1/{userName}/{brainName}`
+
+| Parameter | Description |
+| --- | --- |
+| userName | name of the user who has the BRAIN |
+| brainName | name of the BRAIN |
+
+### Response
+
+| Parameter | Description |
+| --- | --- |
+| id | numerical id of the BRAIN |
+| name | name of the BRAIN |
+| simulators | array of simulations used to train this BRAIN |
+| latest | URL to the latest BRAIN |
+
+
+# Inkling
+
+## GET Inkling
+
+### Request
+
+`GET /v1/{userName}/{brainName}/{brainVersion}/ink`
+
+Inkling code for a BRAIN.
+
+## POST Inkling
+
+Uses the POST request method to post a new version of the Inkling code for a
+BRAIN. You cannot POST new Inkling while a BRAIN is training.
+
+### Request
+
+`POST /v1/{userName}/{brainName}/ink`
+
+#### Headers
+| Header | Description |
+| --- | --- |
+
+| Content‐Type | text/x‐bonsai‐inkling |
+| Content‐Length | size of inkling file in bytes |
+
+#### Body
+The text of the Inkling file.
+
+
+# Training Mode
+
+## PUT Training Mode
+
+Start or stop training mode.
+
+`PUT /v1/{userName}/{brainName}/train`
+
+| Parameter | Description |
+| --- | --- |
+| userName | name of the user who has the BRAIN |
+| brainName | name of the BRAIN |
+
+# Simulators
+
+## GET Simulator Information
+
+> Example Response
+
+```json
+{
+   "name": "breakout",
+   "connected": 2,
+   "instances": [{
+       "state": "ready_to_play".
+       "episode": 1
+   }]
+}
+```
+
+```protobuf
+// There is no protobuf version of this response.
+```
+
+Information for a simulator connected to a BRAIN.
+
+### Request
+
+`GET /{userName}/{brainName}/sims`
+
+| Parameter | Description |
+| --- | --- |
+| userName | name of the user who has the BRAIN |
+| brainName | name of the BRAIN |
+
+### Response
+
+| Parameter | Description |
+| --- | --- |
+| name | name of the simulator |
+| connected | count of how many simulators are connected |
+| instances | array of connected simulators with their status and episode count |
+
+
+## GET Websocket
+
+Upgrade to a Websocket
+
+### Request
+
+`GET /V1/{userName}/{brainName}/sims/ws`
+
+| Parameter | Description |
+| --- | --- |
+| userName | name of the user who has the BRAIN |
+| brainName | name of the BRAIN |
+
+#### Headers
+
+| Header | Value |
+| --- | --- |
+| Upgrade | websocket |
+| Connection | upgrade |
+
+### Response
+
+#### Headers
+
+| Header | Value |
+| --- | --- |
+| Upgrade | websocket |
+| Connection | upgrade |
+
+# Websocket Messages
```

Comment:
In general on the site we're trying not to have sections on the left pane without any text. Even if it's just a simple description of what the code on the right is doing.